lib/runtime/console.js

```diff
@@ -264,7 +262,10 @@ function handleLink (event, uri) {
         pending: atom.config.get('core.allowPendingPaneItems')
       })
     } else {
-      urimatch = uri.match(/([^\:]+)(?:\:(\d+))?/)
+      const matchregex = isWindows ?
+        /(([a-zA-Z]\:)?[^\:]+)(?:\:(\d+))?/ :
```

Comment:
Can match for Windows path like `C:\Users\lalala\atom-julia-client\lib\runtime\console.js`